email/notifier_test.go

```diff
@@ -20,7 +20,7 @@ func (t *TestSender) Send(ctx context.Context, subject string, body string, head
 func TestAsyncWorkerNotifier(t *testing.T) {
 
 	resolver := func(ctx context.Context, id string) ([]collector.Receiver, map[string]interface{}, error) {
-		return []collector.Receiver{}, nil, nil
+		return []collector.Receiver{collector.Receiver{EMail: "user1@test.com", FullName: "User1"}}, nil, nil
```

Comment:
this test need to call [TestSender.Send()](https://github.com/fabric8-services/fabric8-notification/pull/82/files#diff-afc5269574e9814c711df855811b14acR17) and for that we need to return at least one receiver so that zero_receiver check should not skip Send() call.